dashboard/test/ui/README.md

```diff
@@ -47,6 +47,17 @@ You can find the values for these settings in your saucelabs account settings (`
 
 If you want to run tests on Sauce Labs against localhost you need to set up your tunnel:
 
+#### Latest version of Sauce Labs tunnel
```

Comment:
It might be worth mentioning which version these instructions were written for, so that if a newer version is released, we can double-check whether the instructions still work?